# HG changeset patch
# User Felix Janda <felix.janda@posteo.de>
# Date 1428142217 -7200
#      Sat Apr 04 12:10:17 2015 +0200
# Node ID c04cc88f3bb26e254356df6de06583a542ae6796
# Parent  90adc073cbc6398f4fab9b1cd07b685368559f34
# https://bugzilla.mozilla.org/show_bug.cgi?id=1157850
updater: no fts.h and MAXNAMLEN

diff -r 90adc073cbc6 -r c04cc88f3bb2 toolkit/mozapps/update/common/updatedefines.h
--- a/mozilla/toolkit/mozapps/update/common/updatedefines.h	Thu Apr 02 08:10:00 2015 -0400
+++ b/mozilla/toolkit/mozapps/update/common/updatedefines.h	Sat Apr 04 12:10:17 2015 +0200
@@ -109,7 +109,7 @@
 # include <sys/wait.h>
 # include <unistd.h>
 
-#ifdef SOLARIS
+#if 1
 # include <sys/stat.h>
 #else
 # include <fts.h>
diff -r 90adc073cbc6 -r c04cc88f3bb2 toolkit/mozapps/update/updater/updater.cpp
--- a/mozilla/toolkit/mozapps/update/updater/updater.cpp	Thu Apr 02 08:10:00 2015 -0400
+++ b/mozilla/toolkit/mozapps/update/updater/updater.cpp	Sat Apr 04 12:10:17 2015 +0200
@@ -3338,7 +3338,8 @@
   return rv;
 }
 
-#elif defined(SOLARIS)
+#elif 1
+//#defined(SOLARIS)
 int add_dir_entries(const NS_tchar *dirpath, ActionList *list)
 {
   int rv = OK;
@@ -3346,7 +3347,7 @@
   NS_tchar foundpath[MAXPATHLEN];
   struct {
     dirent dent_buffer;
-    char chars[MAXNAMLEN];
+    char chars[NAME_MAX];
   } ent_buf;
   struct dirent* ent;
 
